src/components/admin/SpaceEditForm.tsx

```diff
@@ -464,6 +467,50 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
               </>
             )}
 
+          {space.template === VenueTemplate.conversationspace && (
+            <div>
+              <Toggle
+                label="Show content"
+                register={register}
+                name="showContent"
+                checked={values.showContent}
+              />
+              <InputGroup title="Livestream URL" ml="4">
```

Comment:
I think adding positioning props inside the most generic components might be a headache down the road. It is easier to maintain them if the parent is responsible for the layout i.e. positioning its children without them needing to be aware of it.

That being said, we also have an `<InputGroup />` that acts as such parent, so maybe  it's a good place to put those concerns into it instead of a `<Checkbox />` for example.

```suggestion
              <InputGroup title="Livestream URL" margin="wide" >
```

What other `variant`s of `InputGroup` would we need? Maybe there can be a handful of these layout oriented components.

Reply:
removed this prop from `checkbox` leaving it for `InputGroup` only [b38d47c](https://github.com/sparkletown/sparkle/pull/2930/commits/b38d47c4384bcd7ef41c720aa72ca334965f5741)